build-provider.ps1

```diff
@@ -0,0 +1,56 @@
+<# Build NuGetProvider to be used immediately by PackageManagement #>
+param(
+    # Example: C:\code\OneGet
+    [Parameter(Mandatory=$true)]
+    [string]
+    $OneGetRepositoryRoot,
+
+    [Parameter(Mandatory=$false)]
+    [string]
+    [ValidateSet('net451', 'netcoreapp2.0', 'netstandard1.6', 'all')]
+    $Framework = 'net451',
+
+    [Parameter(Mandatory=$false)]
+    [string]
+    $CopyTo
+)
+
+.\Generate-Resources.ps1
```

Comment:
>. [](start = 0, length = 1)

Consider $PSScriptRoot here as invoking this script with full path from a different working directory will not be able to find this file.

Reply:
Changing to $PSScriptRoot

---
In reply to: [154492010](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154492010) [](ancestors = 154492010)